src/styles/bloks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  background-color: $color-white;
+  border: 1px solid #F3F3F3;
```

Comment:
please use variable for setting color

Reply:
Nie widzę sensu robienia tego w tym zadaniu, ponieważ tego koloru używam tylko raz